docs/geedocs/5.3.2/sources/answer/releaseNotes/relNotesGEE5_3_2.rst.txt

```diff
@@ -78,9 +85,19 @@ avoiding this problem.  1281  Quitting 32-bit Windows portable server
 in Windows 7 via Internet Explorer does not work  IE appears to
 perform aggressive caching, which was preventing the GET request to
 cmd=quit from making it to the server. Changing the command to be a
-POST request prevents this.  1394  Upgrade PostgreSQL and PostGIS to
-latest patch releases  The current patch releases have been
-incorporated.
+POST request prevents this.  1376  Portable Globe Cutter fails the
```

Comment:
What's up with this file? Why did it lose all its formatting? Is it auto generated?

Reply:
rst.txt in sources directory is rst with formatting and tags removed. This is used while displaying the search results. The files which we should review are *.rst files in docsrc folder.  Files in 5.3.2 folder are all generated.